src/content/developers/docs/intro-to-ether/index.md

```diff
@@ -13,45 +13,41 @@ To help you better understand this page, we recommend you first read [Introducti
 
 A cryptocurrency is a medium of exchange secured by a blockchain-based ledger.
 
-A medium of exchange is anything that is widely accepted as payment for goods and services, and a ledger is a data store that keeps track of transactions. Blockchain technology allows users to make transactions on the ledger without reliance upon a trusted third party to maintain the ledger.
+A medium of exchange is anything widely accepted as payment for goods and services, and a ledger is a data store that keeps track of transactions. Blockchain technology allows users to make transactions on the ledger without reliance upon a trusted third party to maintain the ledger.
 
-The first-ever cryptocurrency was Bitcoin, created by Satoshi Nakamoto. Since Bitcoin's release in 2009, people have made thousands of cryptocurrencies across many different types of blockchains.
+The first cryptocurrency was Bitcoin, created by Satoshi Nakamoto. Since Bitcoin's release in 2009, people have made thousands of cryptocurrencies across many different blockchains.
 
 ## What is ether? {#what-is-ether}
 
-Ether (ETH) is the cryptocurrency used to pay for computing services on the [Ethereum blockchain](/developers/docs/intro-to-ethereum).
+**Ether (ETH)** is the cryptocurrency used to pay for computing resources on the Ethereum blockchain.
```

Comment:
"computing resources" is ambiguous to me, because it can be understood both as a compound noun ("This PC is a computing resource") or as a verb phrase ("I am computing some resources" -- whatever that means).

Maybe "computation" would be a better fit here?

_(...)for [the] computation [happening] (on|within) the Ethereum blockchain._

(I need some help with the articles here though)

Reply:
Perhaps just:
```suggestion
**Ether (ETH)** is the cryptocurrency used to pay transaction fees on the Ethereum blockchain.
```
?

Reply:
I honestly think this is too narrow of a definition at this point. Ether is used for much more than just paying transaction fees at this point in time. 
- Its the main currency used when interacting with the NFT market
- Often used as a base currency in collateral for DeFi
- Paying for compute time on the network in the form of gas fees

Im sure there are more than this without much effort to come up with, but I think we should expand this definition more.

Reply:
Yeah, I would agree with this statement. 
Most fundamentally, ETH is required to pay gas, and shortly will be required for network validation and block production (PoS). And yeah, a layer above that, it's used as collateral and as a currency in the evolving metaverse markets. 

Taking a stab at it:
```suggestion
**Ether (ETH)** is the cryptocurrency used for many things on the Ethereum network. Fundamentally, it is the only acceptable form of payment for transaction fees, and after [the merge](/eth2/merge) is also required in order to validate and propose blocks on Mainnet. Ether is also used as a primary form of collateral in the [DeFi](/defi) lending markets, as a unit of account in NFT marketplaces, as payment earned for performing services or selling real-world goods, and more. 
```